index.js

```diff
@@ -0,0 +1,29 @@
+; (() => {
+    const appElement = document.getElementById("app")
```

Comment:
This is element, great.

1. Does this value change everytime? No, ok so should be in global constants